models/lfs.go

```diff
@@ -41,6 +42,12 @@ func (m *LFSMetaObject) Pointer() string {
 	return fmt.Sprintf("%s\n%s%s\nsize %d\n", LFSMetaFileIdentifier, LFSMetaFileOidPrefix, m.Oid, m.Size)
 }
 
+// AsPointer creates a Pointer with Oid and Size
+func (m *LFSMetaObject) AsPointer() *lfs.Pointer {
```

Comment:
I'll have a think about this.

I'm not certain that returning a *Pointer is correct here - it just seems like throwing crap on the heap because we've always done so.

I'm also not certain whether lfsmetaobject should just be implementing an interface or always contain an LFS.Pointer

Reply:
Right, that may be better a normal object. Is there a way for LFSMetaObject to contain a LFS.Pointer struct and use xorm to map the fields to/from the database?

Reply:
yes there should be it might be `xorm: "extends"` but you'd have to check the xorm documentation (and try it)